docs/learn.md

```diff
@@ -28,8 +28,8 @@ The slides are available online here: [slides](http://semicomplete.com/presentat
 
 ## Getting Help
 
-There's [documentation](.) here on this site. If that isn't sufficient, you can
-email the mailing list (logstash-users@googlegroups.com). Further, there is also
+There's [documentation](http://www.elastic.co/guide/en/logstash/current/index.html) here on this site. If that isn't sufficient, you can
```

Comment:
I don't think we should make this change.  If we do this, then folks reading the docs for 1.4.3 will be directed at the docs for `current` (1.5.x), which is not right.

After 1.5.x, we can probably delete docs/learn.md since it's not used in the new docs site, right?